RxExample/RxExample/Examples/GeolocationExample/GeolocationViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  GeolocationViewController.swift
+//  RxExample
+//
+//  Created by Carlos García on 19/01/16.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+#if !RX_NO_MODULE
+    import RxSwift
+    import RxCocoa
+#endif
+
+class GeolocationViewController: ViewController {
+    
+    @IBOutlet weak private var noGeolocationView: UIView!
+    @IBOutlet weak private var button: UIButton!
+    @IBOutlet weak private var button2: UIButton!
+    @IBOutlet weak private var latLabel: UILabel!
+    @IBOutlet weak private var lonLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let geolocationService = GeolocationService.instance
+        
+        geolocationService.autorized
+            .driveNext(driveAutorization)
+            .addDisposableTo(disposeBag)
+        
+        geolocationService.location
+            .driveNext(driveCoordinates)
+            .addDisposableTo(disposeBag)
+        
+        button.rx_tap
+            .bindNext(openAppPreferences)
```

Comment:
I think referencing a method like this would case a retain cycle :(

I believe the same is valid for `driveCoordinates` and `driveAutorization`.

Not sure I have some pretty suggestion if methods are defined on controller. This isn't a problem if extensions are defined on view because then nothing is referencing VC.

¯\\\_(ツ)\_/¯